scripts/library.sh

```diff
@@ -502,6 +502,22 @@ function get_latest_knative_yaml_source() {
   fi
 }
 
+# Returns the latest gke version in [major].[minor].[patch] with provided prefix
```

Comment:
Do you mean

```Returns the latest GKE version in the given [major].[minor] Kubernetes version```

?

Reply:
It's actually intended to return [major].[minor].[patch], see comment below